.github/workflows/build.yml

```diff
@@ -209,7 +213,7 @@ jobs:
       - name: Setup Musl
         if: contains(matrix.container, 'alpine')
         run: |
-          curl -OL https://musl.cc/aarch64-linux-musl-cross.tgz
+          curl -OL https://musl.cc.timfish.dev/aarch64-linux-musl-cross.tgz
```

Comment:
This is musl on your cloudflare worker right? I wonder if we have to self hosted this just to be sure.

Reply:
> This is musl on your cloudflare worker right?

Yep!

I think we only use `aarch64-linux-musl-cross.tgz` in our repositories so we could just host that somewhere?

Reply:
mirrored to https://storage.googleapis.com/sentry-dev-infra-build-assets/aarch64-linux-musl-cross.tgz